### ISSUE TITLE
Gantt drag and drop error

### ISSUE THREAD

**Issue by thijs4ef** [syncfusion/ej2-javascript-ui-controls#115]
Hi! 👋 

Today I used [patch-package](https://github.com/ds300/patch-package) to patch `@syncfusion/ej2-gantt@19.1.66` for the project I'm working on.

In the Gantt I get an error while using the add child option and the row drag and drop feature at the same time:
![gantterror](https://user-images.githubusercontent.com/62025666/121665215-97800c80-caa8-11eb-87ba-362ddbd92929.gif)
In the gif above I forked the Stackblitz gantt demo from the docs and enabled the drag and drop and contextmenu features.
Stackblitz url: https://stackblitz.com/edit/react-aklabb-dmj1wv?file=index.js

Here is the diff that solved my problem:

```diff
diff --git a/node_modules/@syncfusion/ej2-gantt/src/gantt/actions/rowdragdrop.js b/node_modules/@syncfusion/ej2-gantt/src/gantt/actions/rowdragdrop.js
index fef4c25..8095332 100644
--- a/node_modules/@syncfusion/ej2-gantt/src/gantt/actions/rowdragdrop.js
+++ b/node_modules/@syncfusion/ej2-gantt/src/gantt/actions/rowdragdrop.js
@@ -657,7 +657,7 @@ var RowDD = /** @class */ (function () {
                     }
                     childIndex = childRecords.indexOf(deletedRow);
                     flatParentData.childRecords.splice(childIndex, 1);
-                    if (!this.parent.taskFields.parentID) {
+                    if (!this.parent.taskFields.parentID && flatParentData.taskData[this.parent.taskFields.child] !== undefined) {
                         flatParentData.taskData[this.parent.taskFields.child].splice(childIndex, 1);
                     }
                     // collection for updating parent record
```

<em>This issue body was [partially generated by patch-package](https://github.com/ds300/patch-package/issues/296).</em>

**Comment by MonishaSivanthilingam** [comment]
Hi thijs4ef,

Greetings from Syncfusion support.

We were able to replicate the issue you reported and have considered it as a bug from our side. We have logged a bug report for it. You can track its status through the below feedback link.

**Feedback:** https://www.syncfusion.com/feedback/26251/issue-when-drag-and-drop-after-adding-record-through-context-menu

The fix will be provided on the patch release scheduled for June 30, 2021.

We appreciate your patience until then.

Regards,
Monisha.

**Comment by MonishaSivanthilingam** [comment]
Hi thijs4ef,

Thank you for your patience.

The reported issue “[Issue when drag and drop after adding record through context menu](https://www.syncfusion.com/feedback/26251/issue-when-drag-and-drop-after-adding-record-through-context-menu)” have been fixed in version **v19.2.46**. Please find the release notes from the below link.

**Release Notes:** https://ej2.syncfusion.com/documentation/release-notes/19.2.46/?type=all#gantt

Please contact us if you require any further assistance.

Regards,
Monisha.
